src/utils/Processing.ts

```diff
@@ -240,6 +241,26 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+function checkNullValuesOnArray(resource: any): void {
+  const resourceType = resource instanceof InstanceDefinition ? 'Instance' : resource.resourceType;
+  const resourceName =
+    resource instanceof InstanceDefinition ? resource._instanceMeta.name : resource.title;
+  for (const property in resource) {
+    if (!property.startsWith('_') && Array.isArray(resource[property])) {
```

Comment:
I know you're excluding `_` prefixes because they might represent extensions on primitive arrays, in which case `null` is allowed.  But other people (including "future me") might not know that, so can you leave a comment?